SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverAuthorityDebugger.cpp

```diff
@@ -33,6 +33,9 @@ const TArray<FName> USpatialNetDriverAuthorityDebugger::SuppressedProperties = {
 	TEXT("Role"),						// Multiple - all actors
 	TEXT("RemoteRole")					// Multiple - all actors
 };
+const TArray<FName> USpatialNetDriverAuthorityDebugger::SuppressedAutonomousProperties = {
+	TEXT("Pawn"), // Controllers
```

Comment:
Judging by the PR's description, should this be `Role` instead?

Reply:
No that's just me mistyping the PR description - will update